test/runtests.jl

```diff
@@ -45,81 +45,81 @@ AMDGPU.versioninfo()
 
 @info "Running tests with $(length(ws)) workers"
 
-push!(tests, "Core" => ()->include("pointer.jl"))
```

Comment:
Reminder to re-enable these before merge

Reply:
Done